src/sass/shared/mixins.scss

```diff
@@ -132,3 +144,12 @@
         box-shadow: $shadow-color-pill-button-white-hover;
     }
 }
+
+@mixin valign-flex-center($horizontalAlign: false) {
```

Comment:
That's what I'm complaining about, that abstraction for almost the same characters. Vertical centering is the same pain in the ass with or without the mixin if the underlying implementation is the same.

Let's see what @Serabe thinks.